<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewCustomerWindowClass</class>
 <widget class="QMainWindow" name="AddNewCustomerWindowClass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>212</width>
    <height>135</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>212</width>
    <height>135</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Customer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="OkButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CancelButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="TitleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QLabel" name="GivenNamesLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Given Names</string>
    </property>
   </widget>
   <widget class="QLabel" name="LastNameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>LastName</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="GivenNamesTextBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LastNameTextBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="TitleComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Customer's title Mr, Mrs, Ms, Doc</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
